juntagrico/admins/area_admin.py

```diff
@@ -1,7 +1,10 @@
+from django.contrib.admin import register
 from juntagrico.admins import BaseAdmin
+from juntagrico.entity.jobs import ActivityArea
 from juntagrico.util.admin import queryset_for_coordinator
 
 
+@register(ActivityArea)
```

Comment:
dieses pattern werden wir nicht verwenden, ein grosse vorteil von django ist, dass man an spezifischen punkten eine übersicht hat über gewisse konzepte. so hant man an einem punkt die übersicht über alle urls und and einem weiteren punkt über alle entites und mit welchen admin klassen sie registriert sind